src/Controls/src/Core/StackLayout.cs

```diff
@@ -69,17 +69,6 @@ protected override void LayoutChildren(double x, double y, double width, double
 			}
 		}
 
-		// IFrameworkElement Measure
```

Comment:
This code is no longer needed. This exact method was already moved up to the base class